src/types.ts

```diff
@@ -24,7 +24,7 @@ export interface LayerOptions {
 }
 
 export interface MapOptions {
-  topologySource: string;
+  useConfigurationUrl: boolean;
```

Comment:
topologySource is the source for this information now. Delete

Reply:
Removed.